webpack.config.js

```diff
@@ -17,6 +17,12 @@ module.exports = {
         test: /\.(js|jsx)$/,
         exclude: /node_modules/,
         use: 'babel-loader'
+      },
+      {
+        test: /\.(jpg|jpeg|png)$/,
```

Comment:
what is this for?

Reply:
This is need to load the logo png into the html page.
Check this, https://javascript.plainenglish.io/how-to-display-images-from-local-assets-images-folder-when-working-with-react-feb6c5dba526